.github/workflows/ci.yml

```
@@ -0,0 +1,534 @@
+          BREVO_API_KEY=${{ secrets.BREVO_API_KEY }}
+          ADMIN_PASSWORD=${{ secrets.ADMIN_PASSWORD }}
+          ADMIN_SECRET=${{ secrets.ADMIN_SECRET }}
+          EOF
+          echo "✅ Environment variables configured with SQLite database"
+
+      # Database setup removed - SQLite auto-initializes via database service
+      # The database service handles SQLite creation automatically when Turso credentials are not available
+      # No manual setup required for CI environment
+
+      - name: 🚀 Start Vercel Dev Server
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:4:34: Double quote to prevent globbing and word splitting

(shellcheck)